src/Homebrew.jl

```diff
@@ -191,11 +191,11 @@ function update_env()
 end
 
 immutable BrewPkg
-    name::ASCIIString
+    name::String
     version::VersionNumber
-    version_str::ASCIIString
+    version_str::String
     bottled::Bool
-    cellar::ASCIIString
```

Comment:
Assuming no one depends on the implicit conversion here when assigning a `UTF8String`, this change LGTM now.